lib/client.js

```diff
@@ -238,13 +238,7 @@ Client.prototype.useOauth = function(options) {
 Client.create = function(options) {
   options = options || {};
   return new Client(
-      new Dispatcher({
-        asanaBaseUrl: options.asanaBaseUrl,
-        timeout: options.requestTimeout,
-        defaultHeaders: options.defaultHeaders,
-        defaultProxy: options.defaultProxy,
-        logAsanaChangeWarnings: options.logAsanaChangeWarnings
-      }),
+      new Dispatcher(options),
```

Comment:
I thought this change would break developers using `requestTimeout` but it looks like this is already broken since the dispatcher constructor expects `requestTimeout` (not `timeout`).